internal/onprem/hydrate.go

```diff
@@ -0,0 +1,181 @@
+package onprem
+
+import (
+	"context"
+	"encoding/json"
+	"sync"
+
+	"github.com/openshift-assisted/assisted-events-streams/internal/types"
+	"github.com/openshift-assisted/assisted-events-streams/pkg/stream"
+	kafka "github.com/segmentio/kafka-go"
+	"github.com/sirupsen/logrus"
+)
+
+const internalDownloadBuffer int = 1000
+const internalUntarBuffer int = 1000
+const internalEventBuffer int = 1000
```

Comment:
Small: maybe it'd be nice to have a comment about how these numbers were determined?

Reply:
Those were eyeballed but I can set them as env var.